src/ui/base-page-storyEdit.js

```diff
@@ -0,0 +1,299 @@
+                    }
+                }
+            },
+            // the story safety and etiquette notice
+            storyEtiquette: {
+                type: "sjrk.storyTelling.ui",
+                container: ".sjrkc-st-etiquette-container",
+                options: {
+                    components: {
+                        templateManager: {
+                            options: {
+                                templateConfig: {
+                                    templatePath: "%resourcePrefix/templates/etiquette.handlebars"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            // the story preview context
+            storyPreviewer: {
+                type: "sjrk.storyTelling.ui.storyViewer",
+                container: ".sjrkc-st-story-previewer",
+                options: {
+                    selectors: {
+                        progressArea: ".sjrkc-st-story-share-progress",
+                        responseArea: ".sjrkc-st-story-share-response",
+                        responseText: ".sjrkc-st-story-share-response-text"
+                    },
+                    listeners: {
+                        "onStoryViewerPreviousRequested.requestContextChange": "{page}.events.onContextChangeRequested.fire",
+                        "onShareRequested": [{
+                            func: "{that}.showProgressArea",
+                            namespace: "showProgressArea"
+                        },{
+                            func: "{that}.disableShareButton",
+                            namespace: "disableShareButton"
+                        },{
+                            func: "{that}.hideServerResponse",
+                            namespace: "hideServerResponse"
+                        }],
+                        "onShareComplete": [{
+                            func: "{that}.hideProgressArea",
+                            namespace: "hideProgressArea"
+                        },{
+                            func: "{that}.enableShareButton",
+                            namespace: "enableShareButton"
+                        },{
+                            func: "{that}.setServerResponse",
+                            args: ["{arguments}.0"],
+                            namespace: "setServerResponse"
+                        },{
+                            func: "{that}.showServerResponse",
+                            namespace: "showServerResponse"
+                        }]
+                    },
+                    invokers: {
+                        setShareButtonDisabled: {
+                            this: "{that}.dom.storyShare",
+                            method: "prop",
+                            args: ["disabled", "{arguments}.0"]
+                        },
+                        enableShareButton: {
```

Comment:
Eliminate these 6 invokers and lots of the calls to them in favour of mapping this state in model areas - e.g.
progressArea.visible
shareButton.disabled
responseArea.visible
setShareButtonDisabled is an odd man out and should instead be a listener to the change in model state